### ISSUE TITLE
Add cross icon to list pattern

### ISSUE THREAD

**Issue by bethcollins92** [canonical-web-and-design/vanilla-framework#3405]
## Pattern to amend

`.p-list`

## Visual

![image](https://user-images.githubusercontent.com/58959073/98939687-b7832980-24e1-11eb-81a7-82b4685757f6.png)

## Context

I have been asked by a designer to use the cross item in a list, and have seen this being used in more than one place in the code base.

**Comment by bartaz** [comment]
Thanks, we will take it to next WG meeting.

**Comment by lyubomir-popov** [comment]
Validated. Agreed to:

- add an `is-crossed` variant that is styled with a grey cross icon
- keep the current usage of $color-accent for the default tick
- TBD: discuss exact color of `.is-crossed`

**Comment by lyubomir-popov** [comment]
Additional questions raised: 

- Is there a risk of people interpreting the [x] as a close icon? 
- Can it work in a context where there's no ticked icon next to it, e.g. all icons in the list have an [x]
- Somewhat related issue: [x] icon in a maas table

**Comment by bartaz** [comment]
This have been through WG before, but because it needs UX work we should pass it though our new process.

**Comment by clagom** [comment]
Ana is going to collect examples on bullet points vs pro/cons use cases. And the meaning of the colours (e.g. orange)

**Comment by sowasred2012** [comment]
@clagom @anasereijo are there any updates on this? It looks like we're waiting on UX and design decisions before we can move this to dev.

**Comment by clagom** [comment]
@sowasred2012 Thanks for pointing this out, @cristinadresch probably we need to plan for this since last week @anasereijo was away?

**Comment by anasereijo** [comment]
I've done some [design explorations](https://zpl.io/bzxLywG) and we discussed this at the **Design Assembly meeting**.

Key points:
- Let's keep icons as is (tick and cross within circles) for short term solution;
- Brand colour icons don’t work as well - `$color-positive` and `$color-negative` are the preferred colours 
- Explore neutral colours for icons

Questions:
- Do we need new icons? Open a new issue if we decide to go down this route;
- Do we use icons only without them being enclosed in a circle?

**Comment by anasereijo** [comment]
> * Explore neutral colours for icons
> 

@lyubomir-popov 
I suppose this is already in place in [Vanilla ticked list](https://vanillaframework.io/docs/patterns/lists#ticked) with the use of `$color-accent` variable?

**Comment by clagom** [comment]
- Vanilla ticked list is going to be detached from the $color-accent variable?
- The green and red would be the default values for this component

Follow up
- Explore a neutral version, grey component (and specify when to use this neutral version instead of green/red). Opening a different ticket for this

**Comment by anthonydillon** [comment]
Should we abolished the accent concept at this point?

**Comment by lyubomir-popov** [comment]
@anthonydillon after this change it would only be used in the navigation AFAIK. If so then the nav higlight bar colour can follow the structure of other theme colours, which doens't leave anything for the acent color to do. I would vote to deprecate it.
